v0/util/enum/message.js

```diff
@@ -0,0 +1,9 @@
+const Message = {
```

Comment:
The variable name should be more readable. In this repo, `Message/message` refers to a single event. Also, this constant holds the error messages. So a name of that sense should be more appropriate.